fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -15,6 +15,9 @@ class ConfigItem
     # the value which is used if there was no given values and no environment values
     attr_accessor :default_value
 
+    # the value which is used during code generation
```

Comment:
This might require some more comments, what's `code generation`, for new developers it might not be clear it's Swift related

Reply:
I might have added more comments